readme.md

```diff
@@ -97,6 +97,7 @@ try {
 * [opendkim.sig_geterror()](https://github.com/godsflaw/node-opendkim/wiki/opendkim.sig_geterror()): Retrieve the error code associated with a rejected/disqualified signature.
 * [opendkim.sig_geterrorstr()](https://github.com/godsflaw/node-opendkim/wiki/opendkim.sig_geterrorstr()): get the error string specified by the error code.
 * [opendkim.sig_getselector()](https://github.com/godsflaw/node-opendkim/wiki/opendkim.sig_getselector()): get the selector from the signature handle.
+* [opendkim.get_canonlen()](https://github.com/godsflaw/node-opendkim/wiki/opendkim.get_canonlen()): get the canonicalized message length from the signature handle and message.
```

Comment:
this should be `opendkim.sig_getcanonlen()`

Reply:
Not sure why this is still showing up wrong. If you click the file it shows the update though.